posix-configs/eagle/flight/px4.config

```diff
@@ -8,6 +8,7 @@ df_mpu9250_wrapper start
 df_bmp280_wrapper start
 df_hmc5883_wrapper start
 df_trone_wrapper start
+#df_isl_wrapper start
```

Comment:
But I would argue that a user doesn't have two connected at the same time anyway.